src/content/templates/graph-use_vm.json

```diff
@@ -0,0 +1,148 @@
+{
```

Comment:
this is graph-vm the metrics come from the vm plugin. if these are new graphs add them to the existing graph-vm template.